chatkit-android/src/test/kotlin/com/pusher/chatkit/AndroidChatkitDependenciesTest.kt

```diff
@@ -11,17 +11,14 @@ internal class AndroidChatkitDependenciesTest {
     fun `sdkInfo is injected`() {
         val dependencies = AndroidChatkitDependencies(
                 context = stub(),
-                tokenProvider = stub(),
-                okHttpClient = stub(),
-                logger = stub(),
-                platformDependencies = stub()
+                tokenProvider = stub()
         )
 
         assertThat(dependencies.sdkInfo).isEqualTo(SdkInfo(
                 product = "Chatkit",
-                sdkVersion = BuildConfig.VERSION_NAME,
+                sdkVersion = "2.0.0-alpha1",
```

Comment:
what's the intention for this test over time - when we release the SDK would we expect this test to still remain saying `2.0.0-alpha1`, or will we update this with every release?

Reply:
the test should be updated for every release. there was actually a bug that it was [always set to "1.0"](https://github.com/pusher/chatkit-android/pull/195/files#diff-9ab83007d2c24266da686d424538c215L22) (actual SDK version was not taken into account, it wasn't BuildConfig.VERSION_NAME which I removed to remove the confusion)

Reply:
I can see us forgetting to update the test - what will the consequence of that be (will the test start failing?). If it's not possible to extract this out into a string that gets updated automatically, can we start a "release process" doc for v2 and ensure this file gets updated with each release please.

Reply:
Earlier this test unfortunately hasn't found the long-standing bug of the version being reported as "1.0". Normally for the tests it's better when they have their test data as opposed of referencing internal details of the code under test. Redundancy is fine then as it's effectively like double entry system making changes explicit and minimalising chances for error. Also tests then clearly test expected behaviour as opposed to just partially doing so and getting dependent on internal implementation details of the code under test.

Here though I think it's fine to make an exception. It would be inconvenient to have to update the test for every release and probability of introducing a regression here is very low. It seems the original bug was introduced due to the fact that the logic was not correctly designed and never tested (on actual test data and also simulating test going red).

Done: 0e8bb55cfd80fb0debf697b11ce9045f5c754fdf